### DISCUSSION TITLE
Fullscreen HTML editor

### DISCUSSION THREAD

**Discussion by fmenis** [mailcarrierapp/mailcarrier#5] [Ideas]
Hi @danilopolani,
when we are dealing with large templates / layouts, it's a little bit uncomfortable that the edit window cannot be enlarged, or can be opened in a way (another browser tab or a modal) that results comfortable for working with large html.
Maybe something like this can be implemented one day?

**Comment by danilopolani** [comment]
Hey,
thank you for the suggestion! I've renamed the discussion to something more quick to understand. 

Having a button to enlarge the HTML editor is something definitely needed, but right now the project is in a "feature freeze" to work on documentation, bugfixes, and release the version 1.0.

I'm gonna keep this open to work on it as soon as the 1.0 will be released (or even before), thank you again 😄

**Reply by fmenis** [reply]
You're welcome :)
Absolutely understandable, my intention wasn't to request this feature as soon as possible; just give a suggestion.
Byee
